migrations/helpers/dart_samples_update_helper.py

```diff
@@ -136,6 +136,8 @@ def migrate_all_dbs(config: Config, s_start_datetime: str = "", s_end_datetime:
             # create connection to the MLWH database
             with create_mysql_connection(config, False) as mlwh_conn:
                 # 5. update the MLWH (should be an idempotent operation)
+
+                # Check here would migration dbs be ok?
```

Comment:
Hmmm yeah we would need to think through migrations to make sure they are OK with these changes.